ports/nrf/boards/nrf52840_mbr_1M_256k.ld

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Is the MBR used for a bootloader as it is protected? If so, would you know what bootloader is present?

Reply:
Yes, the MBR is a master boot record to use with the bootloader when softdevice is not in use.  Otherwise we'll never able to boot back into bootloader for the non BLE firmware.  The mbr.hex file is included in the softdevice download.  To use the MBR, a special .ld file is require with a firmware start address at 0x1000.